sale_order_revision/view/sale_order.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_sales_order_filter" model="ir.ui.view">
+        <field name="name">sale.order.list.select</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_sales_order_filter" />
+        <field name="arch" type="xml">
+            <field name="partner_id" position="after">
+                <field name="current_revision_id" invisible="1" />
+            </field>
+        </field>
+    </record>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="priority">15</field>
+        <field name="arch" type="xml">
+            <xpath expr="//header" postion='inside'>
```

Comment:
```suggestion
            <xpath expr="//header" position='inside'>
```

Reply:
I've added this in a separate commit for easier backport to v15

Reply:
Although there's a typo, the default is position="inside", so this is not adding a real fix.

Reply:
@pedrobaeza I don't think it is working in v16 by default, we are already using this in our project, and the button was not showing at all, this change fixed the issue we had.

Reply:
OK, maybe OWL is stricter in this way.